rust/src/cli/persist_nic.rs

```diff
@@ -417,3 +438,48 @@ fn has_any_kargs(kargs: &[&str]) -> bool {
         .map(|c| c.split(' ').any(|x| kargs.contains(&x)))
         .unwrap_or_default()
 }
+
+// Preference order on MAC address of bond port:
+// 1. Permanent MAC address
+// 2. MAC address stored in `IFLA_BOND_PORT_PERM_HWADDR`
+fn get_bond_port_mac(iface: &Interface) -> Option<String> {
+    if let Some(mac) = iface.base_iface().permanent_mac_address.as_deref() {
+        return Some(mac.to_string());
```

Comment:
A nit, I noticed that when using the MAC address stored in `IFLA_BOND_PORT_PERM_HWADDR`, we add a log, should we also add a log here before return ?